package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Used only for the development of CAMDEN.GOV.UK Frontend, see `package/package.json` for the published `package.json`",
   "version": "0.4.5",
   "engines": {
-    "node": "~16.14.2"
+    "node": "^20.9.0"
```

Comment:
@oluoluoxenfree Can you bump this to 20.10 inline with the nvmrc file, and check everything still works?